Makefile

```diff
@@ -9,11 +9,11 @@ clean:
 
 build: clean
 	@echo -e "\nBUILDING $(OUTPUT)/juggler BINARY" 
-	mkdir -p ./$(OUTPUT) && go build -o $(OUTPUT)/juggler
+	mkdir -p ./$(OUTPUT) && go build -o $(OUTPUT)/juggler cmd/main.go
```

Comment:
Most tools that I've used (in other languages as well) add command line interfaces to a `bin` directory instead of `cmd`. That's also how most Unixes organize their command lines. Maybe we should adopt these here?

Reply:
This is a Go pattern, it's the way to structure an entry point
https://github.com/golang-standards/project-layout